### ISSUE TITLE
How to resolve list of storage?

### ISSUE THREAD

**Issue by Qsppl** [hybridsjs/hybrids#276]
```TS
const query = {
    type: "search-string",
    string: searchParams.string ?? null,
    rule: searchParams.rule?.id ?? null,
}

// this works, but this use case is not specified in the types
const result = await store.resolve([Project], query)
```

Type of `resolve()` helper:

![image](https://github.com/user-attachments/assets/3cbd9525-1b0b-472f-8341-4bae5695934b)

**Comment by smalluban** [comment]
The types should be updated. I'll do it. Thanks for the issue.

**Comment by smalluban** [comment]
Fixed with dd206112515505b024a35fd2b100a92b21501857
